domains/rem/src/ui/EntityOptionsRow/index.tsx

```diff
@@ -24,7 +24,7 @@ export const EntityOptionsRow: React.FC<EntityOptionsRowProps> = ({
 
 	return (
 		<div className='ee-entity-option__wrapper'>
-			<div className='ee-entity-option__options'>
+			<div className={`ee-entity-option__options ${hideAddNew ? 'ee-entity-option__options--single' : ''}`}>
```

Comment:
let's move where that space is so that it is only added when the modifier is applied

```suggestion
			<div className={`ee-entity-option__options${hideAddNew ? ' ee-entity-option__options--single' : ''}`}>
```

Reply:
Oh I see. I'll update it @tn3rb.